### PR TITLE
AI stuff

### PR DESCRIPTION
**Pull request by emilyaherbert**

### CODE REVIEW

File: game/Assets/Scripts/Classes/Prefab/Shrine.cs [hunk 1]
```diff
@@ -21,6 +21,8 @@ internal class Shrine : MonoBehaviour, IStructure {
     public float SpawnRate => 120.0f;
     [field: SerializeField] public TEAM Team { get; set; }
 
+    public GameObject opposingShrine;
```
Comment (sp1tz) [inline game/Assets/Scripts/Classes/Prefab/Shrine.cs:24]: There is no need for a Shrine to contain this information. Serialize a private field in Minion.cs and set bake it into each prefab Lich/Footman.

File: game/Assets/Scripts/Classes/Prefab/Minion.cs [hunk 2]
```diff
@@ -1,31 +1,122 @@
...
   internal class Minion : MonoBehaviour, ICharacter {
     internal const float START_ACCEL = 10.0f,
                           START_ATTACK = 0.5f,
                           START_DAMAGE = 10.0f,
                           START_HEALTH = 100.0f,
-                          START_MOVING = 10.0f;
+                          START_MOVING = 20.0f;
 
     private NavMeshAgent nav;
+    private int visionRadius = 100;
```
Comment (sp1tz) [inline game/Assets/Scripts/Classes/Prefab/Minion.cs:15]: private const int VISION_RADIUS= 100;

File: game/Assets/Scripts/Classes/Prefab/Shrine.cs (hunk 1 above)
Comment (emilyaherbert) [reply game/Assets/Scripts/Classes/Prefab/Shrine.cs:24]: It's still fine if its done with a public GameObject though I think? If this proves to be an issue later we can go back and write it more carefully.

Comment (sp1tz) [reply game/Assets/Scripts/Classes/Prefab/Shrine.cs:24]: It's not an issue of functionality, it's an issue of design.

**Pull request merged**